extensions/sql-migration/src/scripts/SHIR-auto-configuration.ps1

```diff
@@ -33,7 +33,7 @@ $timeStamp = [System.DateTime]::Now.ToString("yyyyMMddHHmmss")
 $Global:ScriptId = "Script-$timestamp"
 
 # TODO: Auto populate
-$Global:LatestIRVersion = [Version]"5.35.8686.1"
+$Global:LatestIRVersion = [Version]"5.34.8675.1"
```

Comment:
recommend updating this version to 5.37.*.* as well

Reply:
5.34 is the recommended version based on this link [https://go.microsoft.com/fwlink/?linkid=839822]()